packages/init/src/steps/choose-or-create-a-repository.ts

```diff
@@ -127,6 +129,12 @@ export async function chooseOrCreateARepository(
     );
   if (isPreSelectedValid) return preSelectedRepository;
 
+  if (isE2e) {
```

Comment:
Is there another way to do this?

Reply:
like by passing the repo name as an argument?

Reply:
not sure, it's to prevent having the user command line popup to ask the user to enter the name of repo

Reply:
really? if the repo name is available it should create the repo... :/

Reply:
<img width="727" alt="image" src="https://user-images.githubusercontent.com/69144671/183410328-67df0f81-1785-45f9-8133-706cdf4a91b8.png">
that's what I understand from the code yeah? You can pass a `preSelectedRepository` with the flag, and if it has the same name as one in prismic it will be used, otherwise it will show the `await promptForRepoDomain` later

Reply:
ah yes, that's for when --repository is passed down following the instructions,

I guess we could add extra logic to create the repo if it does not exist?

Reply:
don't think we should in my opinion, that would mean we'll change the current behaviour of slice-machine which is not something we have discussed or intended. This way we are only adding logic for the e2e scenario, without touching/changing existing logic